pyproject.toml

```diff
@@ -1,2 +1,63 @@
+# https://packaging.python.org/en/latest/tutorials/packaging-projects/
+# https://hatch.pypa.io/latest/config/build/
+
+[project]
+name = "tutor-xqueue"
+description = "A Tutor plugin for Xqueue (external grading system)"
+authors = [
+  { name = "Edly" },
+  { email = "hello@edly.io"},
+]
+maintainers = [
+  { name = "eduNEXT" },
```

Comment:
shouldn't this be your name and email instead of eduNext @jfavellar90 ?

Reply:
Fixed